ColorMap.cpp

```diff
@@ -0,0 +1,55 @@
+#include "ColorMap.h"
+#include <string>
+
+namespace ColorMapDetails {
+    const char* majorColor[] = { "White", "Red", "Black", "Yellow", "Violet" };
+    const char* minorColor[] = { "Blue", "Orange", "Green", "Brown", "Slate" };
+
+    int printColorMapSize() {
+        int i = 0, j = 0;
+        for (i = 0; i < 5; i++) {
+            for (j = 0; j < 5; j++) {
+                std::cout << i * 5 + j << " | " << majorColor[i] << " | " << minorColor[i] << "\n";
+            }
+        }
+        return i * j;
+    }
+
+    std::string printColorMap() {
+        std::ostringstream colorBuffer;
+        for (int i = 0; i < 5; i++) {
+            for (int j = 0; j < 5; j++) {
+                colorBuffer << (i * 5 + j + 1) << " | ";
+                colorBuffer << majorColor[i] << " | " << minorColor[i] << "\n";
+            }
+        }
+        return colorBuffer.str();
+    }
+
+    std::string expectedColorMap =

```

Comment:
Isn't expectedColorMap part of the test code? It's not needed by the code that prints the manual, right?
Also, make the expected output in a way that would satisfy the user. The user would expect the columns to be aligned?